File: app/src/main/java/in/hbb20/countrycodepickerproject/CustomMasterFragment.java
package in.hbb20.countrycodepickerproject;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.text.Editable;
import android.text.TextWatcher;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

import com.hbb20.CountryCodePicker;


/**
 * A simple {@link Fragment} subclass.
 * create an instance of this fragment.
 */
public class CustomMasterFragment extends Fragment {

    EditText editTextCountryCustomMaster;
    Button buttonSetCustomMaster;
    CountryCodePicker ccp;
    Button buttonNext;

    public CustomMasterFragment() {
        // Required empty public constructor
    }


    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_custom_master_list, container, false);
    }

    @Override
    public void onViewCreated(View view, Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        assignViews();
        editTextWatcher();
        addClickListeners();
    }

    private void addClickListeners() {

        buttonSetCustomMaster.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                String customMasterCountries;
                try {
                    customMasterCountries = editTextCountryCustomMaster.getText().toString();
                    ccp.setCustomMasterCountries(customMasterCountries);
                    Toast.makeText(getActivity(), "Master list has been changed. Tap on ccp to see the changes.", Toast.LENGTH_LONG).show();
                } catch (Exception ex) {

                }
            }
        });


        buttonNext.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                ((ExampleActivity) getActivity()).viewPager.setCurrentItem(((ExampleActivity) getActivity()).viewPager.getCurrentItem() + 1);
            }
        });
    }

    private void editTextWatcher() {
        editTextCountryCustomMaster.addTextChangedListener(new TextWatcher() {
            @Override
            public void beforeTextChanged(CharSequence s, int start, int count, int after) {

            }

            @Override
            public void onTextChanged(CharSequence s, int start, int before, int count) {
                buttonSetCustomMaster.setText("set '" + s + "' as Custom Master List.");
            }

            @Override
            public void afterTextChanged(Editable s) {

            }
        });
    }

    private void assignViews() {
        editTextCountryCustomMaster = (EditText) getView().findViewById(R.id.editText_countryPreference);
        ccp = (CountryCodePicker) getView().findViewById(R.id.ccp);
        buttonSetCustomMaster = (Button) getView().findViewById(R.id.button_setCustomMaster);

        buttonNext = (Button) getView().findViewById(R.id.button_next);
    }
}

File: app/src/main/java/in/hbb20/countrycodepickerproject/CustomMasterFragment.kt
package `in`.hbb20.countrycodepickerproject

import android.os.Bundle
import android.text.Editable
import android.text.TextWatcher
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.Button
import android.widget.EditText
import android.widget.Toast
import androidx.fragment.app.Fragment
import com.hbb20.CountryCodePicker

/**
 * A simple [Fragment] subclass.
 * create an instance of this fragment.
 */
class CustomMasterFragment: Fragment() {

	private lateinit var editTextCountryCustomMaster: EditText
	private lateinit var buttonSetCustomMaster: Button
	private lateinit var ccp: CountryCodePicker
	private lateinit var buttonNext: Button

	override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
		// Inflate the layout for this fragment
		return inflater.inflate(R.layout.fragment_custom_master_list, container, false)
	}

	override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
		super.onViewCreated(view, savedInstanceState)

		assignViews()
		editTextWatcher()
		addClickListeners()
	}

	private fun addClickListeners() {
		buttonSetCustomMaster.setOnClickListener {
			val customMasterCountries: String
			try {
				customMasterCountries = editTextCountryCustomMaster.text.toString()
				ccp.setCustomMasterCountries(customMasterCountries)
				Toast.makeText(activity, "Master list has been changed. Tap on ccp to see the changes.", Toast.LENGTH_LONG).show()
			} catch (ex: Exception) {
			}
		}

		buttonNext.setOnClickListener { (activity as ExampleActivity).viewPager.currentItem = (activity as ExampleActivity).viewPager.currentItem + 1 }
	}

	private fun editTextWatcher() {
		editTextCountryCustomMaster.addTextChangedListener(object: TextWatcher {
			override fun beforeTextChanged(s: CharSequence, start: Int, count: Int, after: Int) {
			}

			override fun onTextChanged(s: CharSequence, start: Int, before: Int, count: Int) {
				buttonSetCustomMaster.text = "set '$s' as Custom Master List."
			}

			override fun afterTextChanged(s: Editable) {
			}
		})
	}

	private fun assignViews() {
		editTextCountryCustomMaster = view!!.findViewById(R.id.editText_countryPreference)
		ccp = view!!.findViewById(R.id.ccp)
		buttonSetCustomMaster = view!!.findViewById(R.id.button_setCustomMaster)

		buttonNext = view!!.findViewById(R.id.button_next)
	}
}
